### DISCUSSION TITLE
`docs/container` does not say to `require('tailwindcss-container-sizes')()` in `tailwind.config.js` plugins

### DISCUSSION THREAD

**Discussion by francoposa** [tailwindlabs/tailwindcss#11853] [Help] [answered]
It appears that the default vanilla-CSS setup does not show how to enable containers?

with` style.css`
```css
@tailwind base;
@tailwind components;
@tailwind utilities;
```

and `tailwind.config.js`
```js
const defaultTheme = require('tailwindcss/defaultTheme');

/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ['./src/**/*.{html,js}'],
  theme: {
    container: {
      center: true,
    },
    extend: {
      fontFamily: {
        sans: ['Inter var', ...defaultTheme.fontFamily.sans],
      },
    },
  },
  plugins: [],
}
```

the container classes are not applied.

The plugin attribute has to be updated to
```js
plugins: [
    require('tailwindcss-container-sizes')(),
  ],
```

but I cannot find this documented anywhere, I just found this old issue that helped me figure it out: https://github.com/tailwindlabs/tailwindcss/issues/446

**Comment by adamwathan** [comment]
Hey! This looks like it's working as expected to me:

https://play.tailwindcss.com/fg3EWW2eEn?file=config

Can you share a demo that reproduces the issue you're having?

**Reply by francoposa** [reply]
full repo that doesn't apply any container class if that plugin is commented out: https://github.com/francoposa/tailwind-demo

**Reply by francoposa** [reply]
it also spits out a warning:

```
% npx tailwindcss -i ./src/css/style.css -o ./src/css/style.tailwind.css --watch

Rebuilding...

warn - No utility classes were detected in your source files. If this is unexpected, double-check the `content` option in your Tailwind CSS configuration.
warn - https://tailwindcss.com/docs/content-configuration
```

**Reply by francoposa** [reply]
and you can see the container class removed from the generated css:

```
diff --git a/src/css/style.tailwind.css b/src/css/style.tailwind.css
index 18fb7aa..b130d97 100644
--- a/src/css/style.tailwind.css
+++ b/src/css/style.tailwind.css
@@ -532,11 +532,4 @@ video {
   --tw-backdrop-opacity:  ;
   --tw-backdrop-saturate:  ;
   --tw-backdrop-sepia:  ;
-}
-
-.container-lg {
-  max-width: 1024px;
-  width: 100%;
-  margin-right: auto;
-  margin-left: auto;
 }
\ No newline at end of file
diff --git a/tailwind.config.js b/tailwind.config.js
index 73f40fc..c5bdc3d 100644
--- a/tailwind.config.js
+++ b/tailwind.config.js
@@ -14,7 +14,7 @@ module.exports = {
     },
   },
   plugins: [
-    require('tailwindcss-container-sizes')(),
+    // require('tailwindcss-container-sizes')(),
   ],
 }
 
lines 1-30/30 (END)
```

**Reply by adamwathan** [reply, marked as answer]
Gotcha, so the issue here is there is no such class as `container-lg` in Tailwind, the class is just `container`. The `container-lg` class looks like something that plugin generates, but Tailwind itself does not include anything like that.

You can see in the documentation that the only class we include is `container`:

https://tailwindcss.com/docs/container

**Reply by francoposa** [reply]
ahhhh okay I see, I guess I confused the breakpoint abbreviations in the table with the class suffixes seen in the other builtins.

Thanks!
